### ISSUE TITLE
Redirects for entire domain

### ISSUE THREAD

**Issue by paavo** [neos/redirecthandler-ui#87]
Is there a way to enable redirects for an entire domain?
For example we'd like to redirect "old" Domains or "dev-domains" to the production URI. 

thanks for a quick hint @Sebobo

**Comment by Sebobo** [comment]
I would recommend using the web server. Easier to configure and better performance.

**Comment by paavo** [comment]
Yes @Sebobo, but we need to redirect Dev-Domains for sites who are published automatically. 
So i prefer to have neos-way 🤔

**Comment by Sebobo** [comment]
I don't fully understand the workflow. Who has the dev domain and needs to be redirected?

**Comment by paavo** [comment]
> I don't fully understand the workflow. Who has the dev domain and needs to be redirected?
Sorry for that @Sebobo 

We have a multi-site Setup in one neos instance:
- Every site is available at a development-domain (ex: siteaaa.devdomain.tld) for adding content
- As soon as the sites production domain (ex. siteaaa.tld) is online, we automatically change the default domain (to siteaaa.tld) for this site and disable the dev domain (siteaaa.devdomain.tld). 
- In this case the dev domain (siteaaa.devdomain.tld) is no longer available and we'd like to setup and automatic redirect to the production domai (siteaaa.tld)

For this case it would be awesome to have a "Neos-Way" to redirect complete Domains (like this is possible in Typo3 for example) or to add redirects  for everything `/*` in Neos Redirects.

**Comment by Sebobo** [comment]
Ok now I understand. This would be a bit of work as all related packages need to be adapted to the change.
Except if we configure this in the `Settings.yaml`. But if I understand you, you want to do this on the fly?

One reason we didn't want to do this yet, was that we wanted to prevent someone from killing their backend by setting up a dangerous redirect. The `/neos/` path would always need to be excluded therefore.

**Comment by paavo** [comment]
A configuration in `Settings.yaml` looks like a good idea. 

>  But if I understand you, you want to do this on the fly?
Yes. We trigger a Neos-Webhook who activates the Domain. This command could also create the "Domain-Redirect i'll guess.
